cas/service.c

```diff
@@ -1913,30 +1913,30 @@ static int cas_device_check(const struct cas_fom   *fom,
 static int cas_id_check(const struct m0_cas_id *cid, const struct cas_fom *fom)
 {
 	const struct m0_dix_layout *layout;
-	int                         rc = 0;
+	int                         rc;
 	struct cas_service         *svc;
-	uint32_t                    device_id = 0;
+	uint32_t                    device_id;
 
+	rc = 0;
 	if (!m0_fid_is_valid(&cid->ci_fid) ||
 	    !M0_IN(m0_fid_type_getfid(&cid->ci_fid), (&m0_cas_index_fid_type,
 						      &m0_cctg_fid_type)))
-		rc = M0_ERR(-EPROTO);
+		return M0_ERR(-EPROTO);
 
 	if (rc == 0 && cas_fid_is_cctg(&cid->ci_fid)) {
```

Comment:
No need to check rc here since assignment was changed to return above.

Reply:
Right. Sorry I missed it.